script.js

```diff
@@ -0,0 +1,43 @@
+const button = document.querySelector('button');
+const title = document.querySelector('.title');
+const author = document.querySelector('.author');
+const content = document.querySelector('.books');
+
+const userData = JSON.parse(localStorage.getItem('formdata')).reverse();
+let count = userData.length;
+
+/* eslint no-unused-vars: "off" */
+function deleteItem(author) {
+  const filteredArray = userData.filter((e) => e.author !== author);
+  localStorage.setItem('formdata', JSON.stringify(filteredArray));
+}
+
+function fetchBooks() {
+  let placeholder = '';
+  userData.forEach((book) => {
+    placeholder += `
+                <div>
+                    <h3>${book.title}</h3>
+                    <p>${book.author}</p>
+                </div>
+                <button onclick="deleteItem('${book.author}')">Remove</button>
+                <hr>
+            `;
+  });
+  content.innerHTML = placeholder;
+
+  button.addEventListener('click', (e) => {
```

Comment:
- [ ] If you can see the error the user will be getting once they try to add a book. and nothing is rendered on the UI as the program crashes. Kindly make sure your functions to implement the core functionalities of the App (`eventListeners`) are declared outside of the function.
![image](https://github.com/ctokoli/Awesome_books/assets/97100091/e9f0a73c-10f9-4e53-b1ad-ac28855eb4ef)